canvas_sdk/data/patient.py

```diff
@@ -0,0 +1,8 @@
+from typing import Optional
+
+from canvas_sdk.data import Model
+
+
+class Patient(Model):
+    id: Optional[str]
```

Comment:
`Optional` is deprecated -- does this work with `str | None`?

Reply:
Yes it does. I think `Optional` is more readable but didn't realize it is deprecated. I can change it back.

Reply:
Done.